op-chain-ops/genesis/config.go

```diff
@@ -761,6 +756,8 @@ type LegacyDeployConfig struct {
 	// DeploymentWaitConfirmations is the number of confirmations to wait during
 	// deployment. This is DEPRECATED and should be removed in a future PR.
 	DeploymentWaitConfirmations int `json:"deploymentWaitConfirmations"`
+
+	UnusedChannelTimeoutGranite uint64 `json:"channelTimeoutGranite,omitempty"`
```

Comment:
A log in a Check function, alike to other non-critical config issues, might be worthwhile. Or maybe an error if it's set to anything else but 0 or the constant global value.